app/services/checksum_validator.rb

```diff
@@ -1,6 +1,15 @@
 # code for validating Moab checksums
 class ChecksumValidator
   include ::MoabValidationHandler
+  include ActiveModel::Validations
+
```

Comment:
I'm not in love with the syntax for the below, but it gets the job done.  Improvements welcome.